### DISCUSSION TITLE
Debian Installation page mentions Cloudsmith, but points elsewhere.

### DISCUSSION THREAD

**Discussion by dariosusman** [rabbitmq/rabbitmq-website#1672] [General]
### Describe the bug

The website https://www.rabbitmq.com/install-debian.html#apt-cloudsmith mentions the use of cloudsmith, but points to  ppa1.novemberain.com which returns 403 for each site.
I've used https://cloudsmith.io/~rabbitmq/repos/rabbitmq-server/setup/#formats-deb for reference.

Best regards,
Dario Susman

### Reproduction steps

1. Go to https://www.rabbitmq.com/install-debian.html#apt-cloudsmith
2. Attempt to use the URLs for Debian APT repos for erlang and rabbitmq-server to fetch required packages

### Expected behavior

Be able to download packages for rabbitmq-server.

**Comment by michaelklishin** [comment]
This is by design. It points to a mirror that does not have a traffic quota that Cloudsmith has.
When the quota is hit, all downloads will be blocked.

You are not supposed to go to `ppa1.novemberain.com`. You are supposed to use it to set up an apt repo
file as demonstrated [in the Debian doc guide](https://rabbitmq.com/install-debian.html),
including a single script that can be executed to set everything up, or a set of steps explained in more details that you can follow one by one.

The repositories on Cloudsmith [are still available](https://cloudsmith.io/~rabbitmq/repos/) and
are used as the source for the mirrors. The mirror even use Cloudsmith account's signing keys.

But I would not recommend using Cloudsmith directly as it may run out of traffic on some months.
